t/t0009-dmesg.t

```diff
@@ -111,5 +111,35 @@ test_expect_success 'logged non-ascii characters handled ok' '
 test_expect_success 'dmesg request with empty payload fails with EPROTO(71)' '
 	${RPC} log.dmesg 71 </dev/null
 '
+test_expect_success 'dmesg -H, --human works' '
+	flux dmesg --human | sed -n 1p \
+	    | grep "^\[[A-Z][a-z][a-z][0-3][0-9] [0-9][0-9]:[0-9][0-9]\]" &&
+	flux dmesg --human | sed -n 2p \
+	    | grep "^\[ *+[0-9]*\.[0-9]*\]"
+
+'
```

Comment:
perhaps add a small comment indicating what the regex is for.  Took me a few seconds to grok intent :-)